docs/System-Requirements.md

```diff
@@ -0,0 +1,21 @@
+# System Requirements and Suggestions
+
+## System requirements
+
+### Memory
+
+OTP is relatively memory-hungry as it includes all the required data in memory. How much memory is required to build the graph for OTP, or to run the OTP server, depends on the used data sets (is OSM, elevation and/or transit data included?), on the size of the covered geographical area and the density of the transit and street network. The required memory can vary from less than one GB to more than 100 GB. For example, including all available data for Finland takes a bit over 10 GB but for Germany it requires 80 GB.
+
+### Processor
+
+Single thread performance is an important factor for OTP's performance. Additionally, OTP benefits from larger CPU cache as reading from memory can be a bottleneck.
+
+OTP's performance scales with the number of available CPU cores. OTP processes each request in a separate thread and depending on the configuration, a single request can use multiple threads. Realtime updates also run in a separate thread. Therefore, to have a good performance, it makes sense to have multiple cores available. How OTP uses parallel processing also depends on the available cores (<= 2 cores vs >2 cores) in some cases. Therefore, load testing should be done against a machine that doesn't differ too much from production machines.
+
+Entur and the Digitransit project have found that the 3rd generation AMD processors have a slightly better performance for OTP2 than the Intel 3rd generation CPUs (and especially better than the 2nd generation CPUs).
```

Comment:
Maybe emphasize that one OTP http request runs for most parts in one thread. There are small operations 
part of an request that can be configured to run in parallel, but in general we do not utilize more than 
one core per request. However, this is something we continuously evaluate and may change.

Reply:
Changed in https://github.com/opentripplanner/OpenTripPlanner/pull/4937/commits/36d0a3c5ff90b1b9b68352ef65cb994a763b10fa